docs/tasks/stateful-sets/deleting-a-statefulset.md

```diff
@@ -0,0 +1,82 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- janetkuo
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to delete a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Deleting a StatefulSet
+
+A Stateful Set can be deleted like other resources in Kubernetes. 
+
+```shell
+kubectl delete -f <file.yaml> 
+```
+
+or
+
+```shell
+kubectl delete statefulsets <statefulset-name>
+```
+
+The associated headless service may need to be deleted separately after the Stateful Set itself is deleted.
+
+```shell
+kubectl delete service <service-name>
+```
+
+Deleting a Stateful Set through kubectl will scale it down to 0, thereby deleting all pods that are a part of it. If you wish to delete just the Stateful Set and not the pods, use `--cascade=false`.
+
+```shell
+kubectl delete -f <file.yaml> --cascade=false
+```
+Setting `--cascade=false` causes the Pods managed by the Stateful Set to be left behind even after the Stateful Set object itself is deleted.
+If the pods have a label `app=myapp`, they can then be deleted as follows.
+
+```shell
+kubectl delete pods -l app=myapp
+```
+
+#### Persistent Volumes
+
+Deleting the pods will *not* delete the volumes. This is to ensure that you have the chance to copy data off the volume before deleting it. Deleting the PVC after the pods have left the [terminating state](/docs/user-guide/pods/index#termination-of-pods) may trigger deletion of the backing Persistent Volumes depending on the storage class and reclaim policy. You should never assume ability to access a volume after claim deletion.
+
+**Note: you may lose all your data once the PVC is deleted, do this with caution.**
+
+#### Complete deletion of a Stateful Set
+
+If you simply want to clean everything in a Stateful Set containing pods which have the label `app=myapp`:
```

Comment:
By the way, I'm pretty sure the concept doc and other docs use "StatefulSet" with no space. Please conform to one style.